suite-native/module-accounts-import/src/screens/AccountImportLoadingScreen.tsx

```diff
@@ -89,28 +90,22 @@ export const AccountImportLoadingScreen = ({
                     }
                     setAccountInfo(fetchedAccountInfo.payload);
                 } else {
-                    showAccountInfoAlert({
-                        title: 'Account info failed',
-                        message: fetchedAccountInfo.payload?.error ?? '',
-                    });
+                    showAccountInfoAlert(fetchedAccountInfo.payload.error, getAccountInfo);
                 }
             }
         };
         try {
             getAccountInfo();
         } catch (error) {
             if (!ignore) {
-                showAccountInfoAlert({
-                    title: 'Account info failed',
-                    message: error?.message ?? '',
-                });
+                showAccountInfoAlert(error?.message ?? '', getAccountInfo);
```

Comment:
I would leave "Account import failed" instead of empty string. What do you think? It's not saying much and we'll be able to debug it quite easily but I guess it's better to fallback to some actual string.

Reply:
fixed in 6bc9ab952